$NetBSD: patch-libmysql_CMakeLists.txt,v 1.1.4.1 2014/04/14 15:29:21 tron Exp $

Use uniform shared library names to avoid packaging list divergence.

--- libmysql/CMakeLists.txt.orig	2014-03-14 18:58:32.000000000 +0000
+++ libmysql/CMakeLists.txt
@@ -207,14 +207,8 @@ IF(NOT DISABLE_SHARED)
     COMPONENT SharedLibraries)
   IF(UNIX)
     # libtool compatability
-    IF(CMAKE_SYSTEM_NAME MATCHES "FreeBSD" OR APPLE)
-      SET(OS_SHARED_LIB_VERSION "${SHARED_LIB_MAJOR_VERSION}")
-    ELSEIF(CMAKE_SYSTEM_NAME MATCHES "HP-UX")
-      SET(OS_SHARED_LIB_VERSION "${SHARED_LIB_MAJOR_VERSION}.0")
-    ELSE()
-      SET(OS_SHARED_LIB_VERSION
+    SET(OS_SHARED_LIB_VERSION
         "${SHARED_LIB_MAJOR_VERSION}.${SHARED_LIB_MINOR_VERSION}.0")
-    ENDIF()
     # Name of shared library is mysqlclient on Unix
     SET_TARGET_PROPERTIES(libmysql PROPERTIES 
       OUTPUT_NAME mysqlclient 
